source/Pacioli/Pacioli.Tests/Tests/JournalTests.cs

```diff
@@ -14,8 +18,43 @@ public void JournalDoesNotAcceptUnbalancedEntries(decimal debit, decimal credit)
             journalEntry.Debits.Add(new JournalEntryItem { Amount = debit });
             journalEntry.Credits.Add(new JournalEntryItem { Amount = credit });
             bool postSuccessful = journal.PostEntry(journalEntry);
-            
+
             Assert.False(postSuccessful);
         }
+
+        [Theory, ClassData(typeof(JournalEntryHasToHaveCreditAndDebit_TestData))]
+        public void JournalEntry_Throws_Exception_On_Empty_Credit_Or_Debit(List<JournalEntryItem> debits, List<JournalEntryItem> credits)
+        {
+            Assert.Throws<ArgumentException>(() => new JournalEntry(debits: debits, credits: credits));
+        }
+
+        private record JournalEntryHasToHaveCreditAndDebit_TestData : IEnumerable<object[]>
+        {
+            public IEnumerator<object[]> GetEnumerator()
+            {
+                yield return new object[]
+                {
+                    new List<JournalEntryItem>(),
+                    new List<JournalEntryItem>()
+                };
+                //yield return new object[]
```

Comment:
Why is this chunk commented out? Does it make the test not pass?

Reply:
It was something that I included initially but then I thought it should have its own test. I really think a JournalEntryItem should have its amount set to a value, and its code to check Amount.

To answer your question, it passes the test. I just excluded it because of my reason above. I shall exclude in the next tests.